gui/.gitignore

```diff
@@ -19,4 +19,6 @@ yarn-error.log*
 *.njsproj
 *.sln
 *.sw?
-.nuxt
\ No newline at end of file
+.nuxt
+yarn.lock
```

Comment:
Sorry but Why do you add lock file here?

Reply:
Unless ones project will be used standalone and not be installed into another project (library), then there's no use in committing any yarn.lock file.

Instead it will always be up to the package.json file to convey what versions of dependencies the project expects then.

Do you want me to restore back it?

Reply:
There are so many discuss about whether to add .lock file to .gitignore. I think we should keep the previous behaviour to prevent unexpected problems.

Reply:
it's okay I'll restore back it